content/blog/2018/egress-tcp/index.md

```diff
@@ -2,65 +2,91 @@
 title: Consuming External TCP Services
 description: Describes a simple scenario based on Istio Bookinfo sample
 publishdate: 2018-02-06
-subtitle: Egress rules for TCP traffic
+subtitle: Mesh-external Service Entries for TCP traffic
 attribution: Vadim Eisenberg
 weight: 92
 aliases:
   - /docs/tasks/traffic-management/egress-tcp/
 keywords: [traffic-management,egress,tcp]
 ---
 
-In my previous blog post, [Consuming External Web Services](/blog/2018/egress-https/), I described how external services can be consumed by in-mesh Istio applications via HTTPS. In this post, I demonstrate consuming external services over TCP. I use the [Istio Bookinfo sample application](/docs/examples/bookinfo/), the version in which the book ratings data is persisted in a MySQL database. I deploy this database outside the cluster and configure the _ratings_ microservice to use it. I define an [egress rule](https://archive.istio.io/v0.7/docs/reference/config/istio.routing.v1alpha1/#EgressRule) to allow the in-mesh applications to access the external database.
+> Note: This blog post was updated on July 23, 2018 to use the new
+[v1alpha3 traffic management API](/blog/2018/v1alpha3-routing/). The old API has been deprecated and will be removed in
+the next Istio release. If you need to use the old version, follow the docs
+[here](https://archive.istio.io/v0.7/blog/2018/egress-tcp.html).
```

Comment:
> Note: This blog post was updated on July 23, 2018 to use the new
+[v1alpha3 traffic management API](/blog/2018/v1alpha3-routing/). 

Delete the rest since old api is gone now.
/

Reply:
Also delete "Node: "